src/pages/ReportDetailsPage.tsx

```diff
@@ -283,32 +305,35 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                             )}
                         </View>
                         <View style={[styles.reportDetailsRoomInfo, styles.mw100]}>
-                            <View style={[styles.alignSelfCenter, styles.w100, styles.mt1]}>
-                                <DisplayNames
-                                    fullTitle={reportName ?? ''}
-                                    displayNamesWithTooltips={displayNamesWithTooltips}
-                                    tooltipEnabled
-                                    numberOfLines={isChatRoom && !isChatThread ? 0 : 1}
-                                    textStyles={[styles.textHeadline, styles.textAlignCenter, isChatRoom && !isChatThread ? undefined : styles.pre]}
-                                    shouldUseFullTitle={shouldUseFullTitle}
-                                />
-                            </View>
-                            {isPolicyAdmin ? (
-                                <PressableWithoutFeedback
-                                    style={[styles.w100]}
-                                    disabled={policy?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}
-                                    role={CONST.ROLE.BUTTON}
-                                    accessibilityLabel={chatRoomSubtitle ?? ''}
-                                    accessible
-                                    onPress={() => {
-                                        Navigation.navigate(ROUTES.WORKSPACE_INITIAL.getRoute(report?.policyID ?? ''));
-                                    }}
-                                >
-                                    {chatRoomSubtitleText}
-                                </PressableWithoutFeedback>
-                            ) : (
-                                chatRoomSubtitleText
+                            {!shouldShowRoomName && (
+                                <View style={[styles.alignSelfCenter, styles.w100, styles.mt1]}>
+                                    <DisplayNames
+                                        fullTitle={reportName ?? ''}
+                                        displayNamesWithTooltips={displayNamesWithTooltips}
+                                        tooltipEnabled
+                                        numberOfLines={isChatRoom && !isChatThread ? 0 : 1}
+                                        textStyles={[styles.textHeadline, styles.textAlignCenter, isChatRoom && !isChatThread ? undefined : styles.pre]}
+                                        shouldUseFullTitle={shouldUseFullTitle}
+                                    />
+                                </View>
                             )}
+                            {!shouldShowRoomName &&
```

Comment:
I think it would be better to combine this with the condition above and then use a `<>`.